packages/calcite-components/src/tests/commonTests/slots.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable jest/no-conditional-expect -- Using conditional logic in a confined test helper to handle specific scenarios, reducing duplication, balancing test readability and maintainability. **/
```

Comment:
Could you look at removing each `eslint-disable` line by adding an overriding `eslintrc.cjs` under `commonTests` where these 2 rules are disabled with an explainer comment for each? I think this should do the trick:

```cjs
module.exports = {
  rules: {
    /* Using conditional logic in a confined test helper to handle specific scenarios, reducing duplication, balancing test readability and maintainability. **/
    "jest/no-conditional-expect": "off",

    /* Util functions are now imported to be used as `it` blocks within `describe` instead of assertions within `it` blocks. */
    "jest/no-export": "off",
  }
}
```

cc @benelan for additional insights